app1.js

```diff
@@ -1,6 +1,6 @@
 const booksSection = document.querySelector('.books');
 const newBooks = [];
-let books = [  
+let books = [
```

Comment:
- [ ] It would be better if you don't have files that you are not using `app1.js`. Please don't commit files that you will not use in your app. Please add this file into `.gitignore` or delete it. 😄 👍